cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-2.launch

```diff
@@ -117,67 +117,67 @@
 
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-1)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="laser_front"/>
-		<param name="topic_name" value="/scan_front"/>
+		<param name="topic_name" value="laser_front/scan_filtered"/>
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-2)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="laser_rear"/>
-		<param name="topic_name" value="/scan_rear"/>
+		<param name="topic_name" value="laser_rear/scan_filtered"/>
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-3)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="laser_top"/>
-		<param name="topic_name" value="/scan_top"/>
+		<param name="topic_name" value="laser_top/scan_filtered"/>
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-4)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="/stereo/left/color_camera"/>
-		<param name="topic_name" value="/stereo/left/image_raw"/>
+		<param name="topic_name" value="stereo/left/image_raw"/>
```

Comment:
I guess we should use `stereo_left` rather than `stereo/left`
I know that this changes it from what it was, but in case we decide to make it consistent....see the naming for the laser scanner